content/en/tracing/troubleshooting/diagnostic_tool.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Using the .NET diagnostic tool for troubleshooting
+kind: documentation
+---
+
+If after installing the .NET tracer your application does not produce traces as expected, you can try running the diagnostic tool for basic troubleshooting. 
+
+The tool can be installed in two different ways:
+
+- Using the .NET SDK by running the command:
+```
+dotnet tool install -g dd-trace
+```
+- Or by direct download by picking the right version [from the release page](https://github.com/DataDog/dd-trace-dotnet/releases)
+**TODO: We probably want to setup a short url?**
```

Comment:
I agree, definitely. We'll link to the standalone tool, right? Will be very useful for support. Not sure how easy it will be to have a stable link though, if we include the version in the tool name (so we probably shouldn't)